content/release-2024-0/announcements-2024-0.md

```diff
@@ -20,6 +20,117 @@ This change affects the following APIs:
+
+Starting from this release we are introducing the support of automated persistence of measurement values under `c8y_LatestMeasurements` fragment.
+
+##### How to enable it
+Using tenant options you should create a category named `measurement.series.latestvalue` with put request to [ tenant options category ](https://cumulocity.com/api/core/#operation/putCategoryOptionResource)
+Example: 
+```
+PUT /tenant/options/measurement.series.latestvalue
+{
+  "c8y_Humidity.H":"", // to enable single series c8y_Humidity.H
+  "c8y_Temperature.*":"", // to able series under fragment c8y_Temperature
+  // or "*":"" to enable all 
+}
+```
+Where the key is a filter of measurement series that should be persistent and value must be always empty string, it's left for future use case.
+
+##### How it works
+
+When measurement is created with series that are matching to configuration then device managed object
+will be updated with last series send to platform.
+E.g. when you send 
+```
+POST /measurement/measurements
+{
+  "source":"5413"
+  "time":"2024-02-01T10:00:00Z"
+  "c8y_Temperature":{
+     "T": {
+        "value": 15,
+        "unit":"C"
+     }
+  }
+  "c8y_Speed":{
+    "S": {
+      "value": 15,
+      "unit":"m/s"
+    }
+  }
+}
+```
+Then if you consider our example configuration only `c8y_Temperature.T` will be stored as part of device,the `c8y_Speed.S` will be ignored.
+Means measurement will be stored as before, only the state update is skipped.
+To read the latest values on device level you need to use inventory api.
+To get single device:
+```
+GET /inventory/managedObjects/5413?withLatestValues=true
+{
+   ...
+   "c8y_LatestMeasurements":{
+        "c8y_Temperature":{
+           "T":{
+             "value":15,
+             "time":"2024-02-01T10:00:00Z",
+             "unit":"C"
+           }
+        }
+   }
+}
+
+```
+or to get list of devices matching expected criteria 
+e.g. get all devices which have reported temperature higher than 10 degrees :
+
+```
+GET /inventory/managedObjects?withLatestValues=true&query=$filter=c8y_LatestMeasurements.c8y_Temperature.T.value+gt+10
+{
+  managedObjects: [
+    {
+        ...
+        "c8y_LatestMeasurements":{
+            "c8y_Temperature":{
+                "T":{
+                    "value":15,
+                    "time":"2024-02-01T10:00:00Z",
+                    "unit":"C"
+                }
+            }
+        }
+    }
+  ]
+}
+
+```
+##### Implications
+
+The feature is introducing addition operation upon measurement creation.
+That will result in performance degradation, which depends on number of series that needs to be 
+stored with each and goes from 5% for single series in each measurement to 
+more than 20% in case of 50 series per measurement.
+
+#### Limitations
+
+##### Security
+
+The latest measurements values are part of managed object, and they follow the managed object inventory role permissions.
+It's not respecting the inventory roles for measurements. 
```

Comment:
Where does "it" refer to?

Reply:
It's a continuation of previous sentence, I'm still unsure how to put words in correct way:  ` they are not respecting the inventory roles for measurements but instead it follows the managed object inventory role permissions` 
Would it be better ?

Reply:
added a suggestion as a separate comment